sensu/plugins/check-keystone-expired-tokens.py

```diff
@@ -80,7 +83,14 @@ def main(cls):
     config_files = None
     if os.path.exists(dev_conf):
         config_files = [dev_conf]
+    
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-c', '--critical',  type=int, default=1000, help='Critical value',  action='store')
+    parser.add_argument('-w', '--warning',  type=int, default=10000, help='Warning value',  action='store')
+    args = vars(parser.parse_args())
+	
+    WATERMARK_WARNING = args['warning']
```

Comment:
this does not work. pass `args.warning` and `args.critical` to the function.

Also, capitalized variables are reserved for constants by convention.  This is not a constant.